client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
It's easy to look up that info though:

https://caniuse.com/#feat=arrow-functions
https://caniuse.com/#feat=serviceworkers

I know it's not binary, but it's fairly safe to assume that if SW is supported in a browser, most of the ES6 features we would want to use are supported. We can webpack it in #1056 (if anyone ever wants to do it...)